packages/components/src/tools-panel/styles.ts

```diff
@@ -148,7 +148,7 @@ export const DropdownMenu = css`
 `;
 
 export const ResetLabel = styled.span`
-	color: var( --wp-admin-theme-color-darker-10 );
+	color: ${ COLORS.ui.borderFocus };
```

Comment:
Along with this change, there are a couple of other pre-existing places in the codebase where this variable is being used for things other than a border... do we want to consider renaming it in `utils/color-values.js` in a followup PR?

Reply:
> Also unclear why this ResetLabel is a darker-10 and not just the theme color 🤔 Do you remember

Thanks for the ping. 

I'm drawing a blank on why that CSS var was actually used. 

As long as the color of the ResetLabel matches the "Saved" in the top toolbar when saving a draft post it will satisfy the request [here](https://github.com/WordPress/gutenberg/issues/44096#issuecomment-1247762119).

Further context and history can be found in https://github.com/WordPress/gutenberg/pull/44260.

Reply:
Was looking at this with @mirka a little and this button doesn't have a disabled state, the 'reset' just isn't displayed when it isn't an option.

Looking at the focused/hovered state, I think I'd recommend we keep the current darker color. If the Reset span is given the same accent color as the rest of the list item, it becomes difficult to differentiate which button you're actually interacting with:

https://user-images.githubusercontent.com/13856531/199766554-d291cdf3-f9e8-4e53-a805-ff1d4f124b6f.mov

With the current darker color, there's at least a little contrast creating some distinction, though it _is_ really subtle. We may want something with more contrast/decoration, but probably not something for this PR.

Any objection to keeping the slightly darker reset for now, using the newly exposed variable?

Reply:
Yeah, we can keep it as is 👍 Thanks for looking into it.